examples/streamed/streamed-server.js

```diff
@@ -1,54 +1,68 @@
-var express = require('express');
+var express = require("express");
 var request = require("request");
 var app = express();
 var port = process.env.PORT || 4444;
-var path = require('path');
+var path = require("path");
 
-var serverManifestJson = require("r2-streamer-js/dist/es5/src/http/server-manifestjson").serverManifestJson;
-var serverMediaOverlays = require("r2-streamer-js/dist/es5/src/http/server-mediaoverlays").serverMediaOverlays;
-var serverPub = require("r2-streamer-js/dist/es5/src/http/server-pub").serverPub;
-var remoteServerPub = require("r2-streamer-js/dist/es5/src/http/server-url").serverRemotePub;
-var serverAssets = require("r2-streamer-js/dist/es5/src/http/server-assets").serverAssets;
-var Server = require("r2-streamer-js/dist/es5/src/http/server").Server
+var serverManifestJson = require("r2-streamer-js/dist/es5/src/http/server-manifestjson")
+  .serverManifestJson;
+var serverMediaOverlays = require("r2-streamer-js/dist/es5/src/http/server-mediaoverlays")
+  .serverMediaOverlays;
+var serverPub = require("r2-streamer-js/dist/es5/src/http/server-pub")
+  .serverPub;
+var remoteServerPub = require("r2-streamer-js/dist/es5/src/http/server-url")
+  .serverRemotePub;
+var serverAssets = require("r2-streamer-js/dist/es5/src/http/server-assets")
+  .serverAssets;
+var Server = require("r2-streamer-js/dist/es5/src/http/server").Server;
 
-var encodeURIComponent_RFC3986 = require("r2-utils-js/dist/es5/src/_utils/http/UrlUtils").encodeURIComponent_RFC3986;
+var encodeURIComponent_RFC3986 = require("r2-utils-js/dist/es5/src/_utils/http/UrlUtils")
+  .encodeURIComponent_RFC3986;
 
 app.use("/viewer", express.static(__dirname + "/../../viewer"));
 app.use("/", express.static(__dirname + "/../../node_modules/requirejs"));
-app.set('views', __dirname);
-app.set('view engine', 'ejs');
+app.set("views", __dirname);
+app.set("view engine", "ejs");
 
 /** This is based on r2-streamer-js' server.ts functionality.
  * However, it only serves the NYPL reader and not the other readers.
  */
 class RemoteEpubServer extends Server {
   constructor() {
     super();
-    this.disableReaders=true;
+    this.disableReaders = true;
     this.publications = [];
     this.pathPublicationMap = {};
     this.expressApp = express();
 
-    this.expressApp.use("/readerNYPL", express.static(__dirname + "/../../viewer"));
+    this.expressApp.use(
+      "/readerNYPL",
+      express.static(__dirname + "/../../viewer", { extensions: ["js"] })
```

Comment:
This change here makes is to so the express will look for missing files with the `.js` extension so if it’s trying to get
http://localhost:4444/readerNYPL/HTMLUtilities and it 404s
it should check for `
http://localhost:4444/readerNYPL/HTMLUtilities.js` and only return a 404 if that is not found